File: Java/FloodTech/src/main/java/com/example/floodtech/dto/LocalizacaoDTO.java
package com.example.floodtech.dto;

import jakarta.validation.constraints.NotBlank;
import lombok.Data;

@Data
public class LocalizacaoDTO {
    private Long id;
    private Double latitude;
    private Double longitude;
    @NotBlank
    private String bairro;
    @NotBlank
    private String cidade;
    @NotBlank
    private String estado;
    @NotBlank
    private String cep;
}

File: .Net/FloodTech.API/FloodTech.Application/DTOs/LocalizacaoDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace FloodTech.Application.DTOs
{
    public class LocalizacaoDto
    {
        public int Id { get; set; }
        public double Latitude { get; set; }
        public double Longitude { get; set; }
        public required string Bairro { get; set; }
        public required string Cidade { get; set; }
        public required string Estado { get; set; }
        public required string Cep { get; set; }
    }
}